backend/tests/integration/records/api/test_api_conferences.py

```diff
@@ -311,3 +311,19 @@ def test_number_of_contributions_query_after_hard_delete(inspire_app):
 
     expected_contributions_number = 0
     assert expected_contributions_number == conference.number_of_contributions
+
+
+def test_cn_redirection_works_for_conferences(inspire_app):
```

Comment:
what is `cn` (appears in the test case many collections)?

Reply:
cn - control number